draft-rescorla-tls-subcerts.md

```diff
@@ -265,6 +281,10 @@ as those noted in [Jager et al.].  So there would be a need to define some marke
 that would be inserted into an end-entity certificate to indicate that it could
 be used to issue sub-certificates.
 
+[[ Nick - Is this really an issue we want to engineer around? Requiring CAs to add
+a new X.509 extension and operators to re-issue all existing certificates will likely
+hamper the adoption of this draft. ]]
```

Comment:
I think that it's a pretty hard requirement that this be opt-in for servers, because otherwise this becomes a threat to existing servers (cf. J\"{a}ger et al.)